src/lib.rs

```diff
@@ -314,17 +314,7 @@ impl<'a, B, A> Doc<'a, B, A> {
         B: Deref<Target = Doc<'b, B, A>>,
         W: ?Sized + fmt::Write,
     {
-        self.render_raw(width, &mut render::FmtWrite(out))
-    }
-
-    /// Writes a rendered document to a `RenderAnnotated<A>` object.
-    #[inline]
-    pub fn render_raw<'b, W>(&'b self, width: usize, out: &mut W) -> Result<(), W::Error>
```

Comment:
Hmm, I did mean for this to be usable though as I want to try and use `RenderAnnotated` to attach links in gluon's documentation. If `Render` and `RenderAnnotated` is just re-exported from the root instead it should all work.